ocean_data_parser/read/vocabularies/seabird_variable_attributes.json

```diff
@@ -1385,7 +1391,8 @@
     },
     "longitude": {
         "long_name": "Longitude",
-        "units": "deg",
+        "bodc_alternative_label": "Lon",
+        "units": "deg east",
```

Comment:
I would standardize that one to the CF recommanded unit `degrees_east`

Reply:
Agreed !

Reply:
I made the changes